hangman.js

```diff
@@ -1 +1,63 @@
-console.log("welcome to hangman");
\ No newline at end of file
+var split;
+var word;
+var guess;
+var numberOfGuess=0;
+// get user word 
+var getWord=function(){
+	word=prompt("Choose a word.").toLowerCase();
+	return word;
+};
+
+var splitWord=function(wordToSplit){ 
+	split=wordToSplit.split("");
+	word=word.split("");
+	
+	console.log("split:", split);
+	
+	for (var i=0; i<split.length; i++){
+		split[i] = "_";
+	}
+	
+	return split;
+};
+
+var guessWord=function(){
+	
+	while(numberOfGuess<word.length){
+		guess=prompt("Please guess a letter to fill in the blank: " + split).toLowerCase();
+		numberOfGuess++;
+		alert(numberOfGuess)
+		
+		for (var i=0; i<word.length; i++){
```

Comment:
Until you get comfortable using more, smaller functions, I recommend more comments.  It would be nice to see a comment indicating what the intent is for this group of code . e.g. "Replace '-' for correct guesses."